modules/Microsoft.Network/firewallPolicies/readme.md

```diff
@@ -13,7 +13,7 @@ This module deploys Firewall Policies.
 
 | Resource Type | API Version |
 | :-- | :-- |
-| `Microsoft.Network/firewallPolicies` | [2021-05-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-05-01/firewallPolicies) |
+| `Microsoft.Network/firewallPolicies` | [2022-01-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/firewallPolicies) |
```

Comment:
Please update to latest version documented in ARR so that the link is mapped to the specific version used and not to generic latest (which can change over time)

Reply:
Updated to older API version